src/components/RuleCard/index.jsx

```diff
@@ -771,6 +778,8 @@ RuleCard.propTypes = {
   onRuleDelete: func,
   // If true, the card will hide all buttons.
   readOnly: bool,
+  // If true, the card will disable all buttons
```

Comment:
I think the revisions button in the rule card doesn't currently get disabled when `disableActions` is `true`. If this is intended, then we should probably mention it in the comment.